packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -295,7 +308,8 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
         };
 
         this.connectionManager = createConnectionManager(props);
-
+        this.mc = loggerToMonitoringContext(logger);
```

Comment:
This is technically a duplicate class variable to `this.logger`, because the MC gets mixed in on the existing logger object.  I think convention is to either ditch `this.logger` (and do `this.mc.logger` throughout), or update `this.logger` to be a property that returns `this.mc.logger`.